crates/cli/src/subcommands/identity.rs

```diff
@@ -152,7 +152,7 @@ fn get_subcommands() -> Vec<Command> {
             ,
         Command::new("remove")
             .about("Removes a saved identity from your spacetime config")
-            .arg(Arg::new("identity")
+            .arg(common_args::identity()
```

Comment:
revert this one

Reply:
(nevermind - we thought this was the only param to `spacetime identity remove`, but it turns out this is not true)